config/deploy/production.rb

```diff
@@ -10,6 +10,35 @@
 
 set :branch, ENV['BRANCH'] || 'master'
 
+def config_map
+  {
+    "catalog-production" => "catalog-production",
+    "catalog-staging" => "catalog-staging",
+    "cicognara" => "cicognara",
+    "dpul" => "dpul",
+    "figgy" => "figgy",
+    "lae" => "lae",
+    "pulmap" => "pulmap",
+    "pulfa3" => "pulfa3",
+    "orangelight" => "orangelight" # reserves and dss use this
```

Comment:
The pulfa3 and orangelight folders don't exist within solr_configs. pulfa3 should be renamed to plantain and we should remove orangelight since there isn't a way to update the configset (unless if we want to do `"catalog-production" => "orangelight"` until we delete the configset).

Reply:
my understanding is that "plantain" won't stick around as the name of that application, so we probably ought to keep it as "pulfa3."

I'll add the other apps on `orangelight` to #170 as per discussion on slack.

Reply:
Okay the orangelight config sets have been cleaned up on the server and removed from this pr.

I think we ought to keep pulfa3; it doesn't seem like plantain is going to stick around beyond next week as a name for that app.

ready for re-review.

Reply:
I think we ought to keep pulfa3 since it doesn't look like plantain will be the name of that app beyond next week.

The config sets are cleaned up on the server and removed from this PR. ready for re-review.

Reply:
Ok, I think we should do `"plantain" => "pulfa3"` or remove it altogether. The deploy fails since the the "pulfa3" directory doesn't exist within [solr_configs](https://github.com/pulibrary/pul_solr/tree/master/solr_configs).

Reply:
Oh! Okay. thanks for catching that.

It looks like on staging solr the `pulfa3` collection uses a configset called `pulfa3-staging`. On prod solr there is no pulfa3 collection, but there are config sets called `pulfa3` and `plantain`.

So maybe rename the `plantain` config directory to `pulfa3-staging` and delete those 2 config sets on prod for now?

Reply:
> I think we ought to keep pulfa3 since it doesn't look like plantain will be the name of that app beyond next week.

I agree with this, and I apologize for this. At the time, I assumed that this was a convention being followed for all configsets.

Reply:
no worries! I made these adjustments. see what you think @tampakis!